ios/brave-ios/Sources/BraveWallet/Crypto/WalletActions/SendTokenView.swift

```diff
@@ -72,75 +77,97 @@ struct SendTokenView: View {
     }
   }
 
+  private var showUnavailableBTCBalanceBanner: Bool {
+    guard keyringStore.selectedAccount.coin == .btc,
+      let balancesForSelectedAccount = sendTokenStore.btcBalances[keyringStore.selectedAccount.id]
+    else { return false }
+    return balancesForSelectedAccount[.pending] ?? 0 != 0
+  }
+
+  private var sendTokenBalanceDisplay: String {
+    guard let selectedSendToken = sendTokenStore.selectedSendToken,
+      let selectedSendTokenBalance = sendTokenStore.selectedSendTokenBalance
+    else {
+      return "0"
+    }
+    return String(
+      format: "%@ %@",
+      selectedSendTokenBalance.decimalExpansion(precisionAfterDecimalPoint: 6)
+        .trimmingTrailingZeros,
+      selectedSendToken.symbol
+    )
+  }
+
+  private var sendTokenRow: some View {
+    Button {
+      self.isShowingSelectAccountTokenView = true
+    } label: {
+      HStack {
+        if let token = sendTokenStore.selectedSendToken {
+          if token.isErc721 || token.isNft {
+            NFTIconView(
+              token: token,
+              network: networkStore.defaultSelectedChain,
+              url: sendTokenStore.selectedSendNFTMetadata?.imageURL,
+              length: 26
+            )
+          } else {
+            AssetIconView(
+              token: token,
+              network: networkStore.defaultSelectedChain,
+              length: 26
+            )
+          }
+        }
+        VStack(alignment: .leading) {
+          Text(sendTokenStore.selectedSendToken?.symbol ?? "")
+            .font(.title3.weight(.semibold))
+            .foregroundColor(Color(.braveLabel))
+          Text(networkStore.defaultSelectedChain.chainName)
+            .font(.caption)
+            .foregroundColor(Color(.secondaryBraveLabel))
+        }
```

Comment:
Can these get to multiple lines? May want to add `multilineTextAlignment(.leading)`, cant remember what the default is